src/main/java/mate.academy/Main.java

```diff
@@ -3,14 +3,25 @@
 import java.util.List;
 import mate.academy.lib.Injector;
 import mate.academy.model.Product;
+import mate.academy.service.FileReaderService;
+import mate.academy.service.ProductParser;
 import mate.academy.service.ProductService;
 
 public class Main {
 
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+
+        FileReaderService fileReaderService =
```

Comment:
Please use the @Inject annotation to automatically inject the FileReaderService instance instead of directly using the Injector.